app/controllers/agenda-items/index.ts

```diff
@@ -247,6 +276,11 @@ const agendaItemsQuery = ({
     filters[':query:abstract_location_id'] = queryIds;
   }
 
+  // Apply optional filter for governing body labels
+  if (governingBodyClassifications) {
+    filters['governing_body_classification'] = governingBodyClassifications;
```

Comment:
**Blocker:** As discuss during the standup, it should filter on id, instead of name